### DISCUSSION TITLE
Support Google Cast

### DISCUSSION THREAD

**Discussion by FlyinPancake** [LeoKlaus/plappa#218] [Feature Requests]
I love the app a lot, and the only thing missing is Google Cast support. 

It is documented here <https://developers.google.com/cast/docs/ios_sender/integrate>

**Comment by FlyinPancake** [comment]
Here is a sample app <https://github.com/googlecast/CastVideos-ios>

**Comment by LeoKlaus** [comment]
Hi FlyinPancake, thank you for the suggestion!

There already is a discussion for Google Cast support at #110, though it only shows up when you search for `chromecast` in the discussions section.

The status hasn't really changed since then, I'd like to implement support for Google Cast but it's not a very high priority.

I'll close this discussion and post eventual updates to #110.
